### DISCUSSION TITLE
[Support]: min_initialized - how does it work?

### DISCUSSION THREAD

**Discussion by jiiins** [blakeblackshear/frigate#11370] [General Support]
### Describe the problem you are having

I would like to make sure that an object has been identified in a few frames before triggering an event (to reduce false positives due to raindrops). The definition of  `min_initialized` says 'Number of consecutive detection hits required for an object to be initialized in the tracker.'

If the parameter is set to 5 for example, does this mean that an object will need to be detected in 5 consecutive frames? Will the HA MQTT notification fire after that?

Or is there another way?

Thanks

### Version

0.13.2-6476F8A

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

n/a

**Comment by NickM-27** [comment]
> If the parameter is set to 5 for example, does this mean that an object will need to be detected in 5 consecutive frames?

More or less, it's a bit more complicated than that but that's the gist
